nodejs/server.js

```diff
@@ -130,7 +130,7 @@ app.post("/specialize", withEnsureGeneric(specialize));
 app.post("/v2/specialize", withEnsureGeneric(specializeV2));
 
 // Generic route -- all http requests go to the user function.
-app.all("/", function(req, res) {
```

Comment:
I think we need to check if this is backward compatible and does not break any of existing functionality.

Reply:
Yes, change is backwards compatible. It doesn't break existing functionality.